test/tests/ver-1ha/comparison_ver-1ha.py

```diff
@@ -0,0 +1,136 @@
+import csv
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+from scipy import special
+import os
+import math
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+os.chdir(os.path.dirname(__file__))
+
+# ===============================================================================
+# Extract TMAP8 results
+tmap8_sol = pd.read_csv("./gold/ver-1ha_out.csv")
+tmap8_sol_time = tmap8_sol['time']
+tmap8_sol_P2 = tmap8_sol['P2_val']
+tmap8_sol_P3 = tmap8_sol['P3_val']
+tmap8_sol_C2 = tmap8_sol['C2_val']
+tmap8_sol_C3 = tmap8_sol['C3_val']
+
+
+# ===============================================================================
+# Calculate analytical solution
+def get_analytical_solution(t_vect):
+    """Returns the pressure and concentration for the 2nd and 3d enclosures as a function of time
+    Args:
+        t_vect (ndarray): time in s
+    Returns:
+        P2 (np.array): pressure of T3 in the 2nd enclosure in Pa
+        P3 (np.array): pressure of T3 in the 3rd enclosure in Pa
+        C2 (np.array): concentration of T3 in the 2nd enclosure in atoms/m^3
+        C3 (np.array): concentration of T3 in the 3rd enclosure in atoms/m^3
```

Comment:
```suggestion
        P2 (np.array): pressure of T2 in the 2nd enclosure in Pa
        P3 (np.array): pressure of T2 in the 3rd enclosure in Pa
        C2 (np.array): concentration of T2 in the 2nd enclosure in atoms/m^3
        C3 (np.array): concentration of T2 in the 3rd enclosure in atoms/m^3
```
Am I missing something?